central/policy/datastore/datastore_impl_postgres_test.go

```diff
@@ -187,7 +185,7 @@ func (s *PolicyPostgresDataStoreTestSuite) TestImportPolicy() {
 }
 
 func (s *PolicyPostgresDataStoreTestSuite) TestSearchPolicyCategoryFeatureDisabled() {
```

Comment:
nit:  I'm not sure this test is valid as this is explicitly a Postgres test.  It doesn't cause harm but it essentially just tests RocksDB functionality which would be covered in the RocksDB backed tests.

Reply:
Not really, a good test to have since policy object doesn't get touched in the rocksdb path.